ttyplot.c

```diff
@@ -370,6 +373,17 @@ int main(int argc, char *argv[]) {
 
     redraw_screen(errstr);
 
+    // If stdin is redirected, open the terminal for reading user's keystrokes.
+    int tty = -1;
+    if (!isatty(STDIN_FILENO))
+        tty = open("/dev/tty", O_RDONLY);
+    if (tty != -1) {
+        // Disable input line buffering and echo. The functions below work even when
+        // stdin is redirected: they search for a terminal in stdout and stderr.
+        cbreak();
+        noecho();
```

Comment:
There is a call to `noecho();` already a few lines up in line 368. Does the first call render the second unecessary or is there a reason to have a second call that I am missing?

Reply:
It's me: I did not notice the previous call, despite it being pretty close... Fixed, force-pushed.

Reply:
@edgar-bonet thanks!